packages/react/src/patterns/sections/CTASection/__stories__/CTASection.stories.js

```diff
@@ -0,0 +1,69 @@
+import './index.scss';
+import { select, text, object, withKnobs } from '@storybook/addon-knobs';
+import CTASection from '../CTASection';
+import React from 'react';
+import readme from '../README.md';
+import { storiesOf } from '@storybook/react';
+
+storiesOf('Patterns (Sections)|CTASection', module)
+  .addDecorator(withKnobs)
+  .addParameters({
+    readme: {
+      sidebar: readme,
+    },
+  })
+  .add('Default', () => {
+    const types = ['local', 'external'];
+    const ctaProps = {
+      style: 'button',
+      type: types[0],
+      buttons: [
+        {
+          type: select('ContentBlock | CTA type', types, types[0]),
```

Comment:
one thing, I think we want to follow the same format for other stories where we write the actual prop name in parenthesizes like so:

'Content Block CTA (required(?)) (cta.type):'